### ISSUE TITLE
Component - Accordion - Not following Wai-aria design pattern

### ISSUE THREAD

**Issue by MeD-Owner** [MeD-DMC/Adapt-accessibilityfixes#14]
Not following the wai-aria design pattern for accordions. Missing the aria-controls attribute.

**Comment by MeD-Owner** [comment]
No issues with Accordions came up during RDI testing! Closing issue
